shopelectro/management/commands/update_in_pack.py

```diff
@@ -0,0 +1,29 @@
+import logging
+
+from django.core.management.base import BaseCommand
+from django.db import transaction
+
+from shopelectro.models import TagGroup
+
+logger = logging.getLogger(__name__)
+
+
+def update():
+    uuid = 'ae30f766-0bb8-11e6-80ea-02d2cc20e118'
```

Comment:
only update_db should know this uuid as isolated place working with awful xml files.
If you move this code to update_db, the problem will gone